### DISCUSSION TITLE
How to include web-based or youtube-based documents in my knowledge base?

### DISCUSSION THREAD

**Discussion by arijit1201** [open-webui/open-webui#6333] [General]
Similar to how notebooklm of google enable us to add sources like web urls or youtube urls as part of the notebook - does open web ui support similar features? I know we can use #<web-url> to use RAG in the chat, but my question is more about the freedom of enhancing my knowledge-base by including website/youtube links to it. If not, I would love to work on the implementation of it!

**Comment by juananpe** [comment]
@arijit1201 , @tjbck what do you think about [this](https://github.com/open-webui/open-webui/pull/8231) approach?
